rationale.md

```diff
@@ -553,7 +553,7 @@ async run_inline_query(
 
 #### Binary responses
 
-A method that only has binary responses will have the request configured for binary inside the generated SDK method. Images and PDFs are considered binary.
+A method that only has binary responses will have a note that the response is binary.
```

Comment:
Still relevant?

Reply:
Yes, we still generate the note into the method header to call out that the response will be binary